example_async.py

```diff
@@ -35,16 +49,26 @@
 def run():
     # Assign configuration variables.
     # The configuration check takes care they are present.
-    api_factory = APIFactory(sys.argv[1])
-    with open('gateway_psk.txt', 'a+') as file:
-        file.seek(0)
-        psk = file.read()
-        if psk:
-            api_factory.psk = psk.strip()
-        else:
-            psk = yield from api_factory.generate_psk(sys.argv[2])
+    conf = load_json(CONFIG_FILE)
+
+    try:
```

Comment:
@lwis these lines (or something along the line of these lines) will appear in all examples. Should I extract this to a class instead?

Reply:
Hmm, we risk making the examples too complex if we make reference to other files. And this doesn't really belong in the main lib. Not sure!

Reply:
Agree and was unsure myself, hence reaching out. Let's keep it as is for now.